indexing-service/src/main/java/io/druid/indexing/common/task/IndexTask.java

```diff
@@ -141,9 +141,19 @@ static RealtimeTuningConfig convertTuningConfig(ShardSpec spec, IndexTuningConfi
 
   private final ObjectMapper jsonMapper;
 
+  public IndexTask(
```

Comment:
Or you can just comment that its only used in unit tests. Whichever makes it easier to read.